interview-questions.md

```diff
@@ -1,63 +1,53 @@
 # ASSESSMENT 4: Interview Practice Questions
+
 Answer the following questions.
 
 First, without external resources. Challenge yourself to answer from memory.
 
-Then, research the question to expand on your answer. Even if you feel you have answered the question completely on your own, there is always something more to learn. Write your researched answer in your OWN WORDS.  
+Then, research the question to expand on your answer. Even if you feel you have answered the question completely on your own, there is always something more to learn. Write your researched answer in your OWN WORDS.
 
 1. What is an instance variable in Ruby? How is it different from other variables?
 
-  Your answer:
-
-  Researched answer:
-
+Your answer:
 
+Researched answer:An instance variable in ruby has a name starting with @ symbol, and its content is restricted to whatever the object itself refers to. Two separate objects, even though they belong to the same class, are allowed to have different values for their instance variables.
 
 2. What is a block in Ruby?
 
-  Your answer:
-
-  Researched answer:
-
+Your answer: its a specific line of code that call only be called in that specified area
 
+Researched answer:A Ruby block is a way of grouping statements, and may appear only in the source adjacent to a method call; the block is written starting on the same line as the method call's last parameter (or the closing parenthesis of the parameter list). The code in the block is not executed at the time it is encountered.
 
 3. Ruby has an implicit return. What does that mean?
 
-  Your answer:
-
-  Researched answer:
-
+Your answer:
```

Comment:
where are your answers?

Reply:
main point of this exercise is to try to develop your own language for describing technical concepts. Practice with trying to provide your own answer first to exercise that muscle of looking for words. It's similar to practice speaking a new language versus just reading in a new language. They're completely different muscle and both need to be exercised.